spec/openhab/dsl_spec.rb

```diff
@@ -483,7 +483,7 @@
         expect(2 * w).to eql 10 | "W"
         expect((2 * kw).to_i).to eq 10_000
         expect(2 * w == 10).to be true
-        expect(5 / w).to eql 1 | "W"
+        expect(5 / w).to eql 1 | "/W"
         expect((2 * w / 2)).to eql w
```

Comment:
You need to add a not_to eql case for when it's false

Reply:
I can't think of a good not_to eql check to add, so I added a check against `kw`. Do you have a better idea?